footer.php

```diff
@@ -1,4 +1,53 @@
-    <footer id="footer">
+  <?php
+    $facebook = IGV_get_option('_igv_socialmedia_facebook_url');
+    $twitter = IGV_get_option('_igv_socialmedia_twitter');
+    $instagram = IGV_get_option('_igv_socialmedia_instagram');
+
+    $contact_id = get_id_by_slug('contact'); 
+    $shipping_id = get_id_by_slug('shipping');
+  ?>
+    <footer id="footer" class="container">
+      <div class="row">
+        <div class="col col3">
+          <?php if (!empty($facebook) || !empty($twitter) || !empty($instagram)) { ?>
+          <ul class="u-inline-list">
+            <?php if (!empty($facebook)) { ?>
+            <li>
+              <a href="<?php echo $facebook; ?>"><img src="facebook"></a>
+            </li>
+            <?php } if (!empty($instagram)) { ?>
+            <li>
+              <a href="https://instagram.com/<?php echo $instagram; ?>"><img src="facebook"></a>
+            </li>
+            <?php } if (!empty($twitter)) { ?>
+            <li>
+              <a href="https://twitter.com/<?php echo $twitter; ?>"><img src="facebook"></a>
+            </li>
+            <?php } ?>
+          </ul>
+          <?php } ?>
+        </div>
+        <div class="col col6 text-align-center font-small-caps">
+          &copy; <?php echo date('Y'); ?> The Corner LLC. All Rights Reserved
```

Comment:
is The Corner LLC really the company name? Maybe that line should be site option too?